modules/customize-certificates-understanding-service-serving.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="understanding-service-serving-{context}"]
+= Understanding service serving certificate secrets
+
+Service serving certificate secrets are intended to support complex 
+middleware applications that need out-of-the-box certificates. It has 
+the same settings as the server certificates generated by the 
+administrator tooling for nodes and masters.
```

Comment:
I'm not sure if this comment about administrator tooling still applies for 4.0. 

It would be better for this last sentence to say that the service serving certificates are issued as TLS Web server certificates.

Reply:
The last sentence has been reworded to just say:

> These certificates are issued as TLS web server certificates.